src/composables/useLogRoutes.ts

```diff
@@ -23,11 +23,11 @@ export const useLogRoutes = (routes: ComputedRef<RouteItem[]>) => {
   const setInputsFromRoute = (routes: RouteItem[]): void => {
     if (routes.length === 1) {
       action.value = routes[0].inputType || 'input-number';
-      distance.value = routes[0].distance || 0;
+      distance.value = routes[0].distance || '0.00';
```

Comment:
String literal `'0.00'` repeat across code > 1, you can use `const`, please.

Reply:
Refactored into global config.